cdap-cli/src/test/java/co/cask/cdap/cli/util/table/TableRendererTest.java

```diff
@@ -78,4 +78,17 @@ public void testTwoLineCell2() {
       .build();
     getRenderer().render(System.out, table);
   }
+
+  @Test
+  public void testLongHeader() {
```

Comment:
How do we actually make sure that the header stays in the table? I don't see any assertions? Is it just a visual test?